src/components/visualizations/chart/highcharts/test/getZeroAlignConfiguration.spec.ts

```diff
@@ -205,31 +211,155 @@ describe("getZeroAlignConfiguration", () => {
         });
     });
 
+    describe("getMinMaxInfo with some null data", () => {
```

Comment:
should not this be as child describe inside `describe("getMinMaxInfo", () => {`?

Reply:
done, please check my latest commit, @ivan-nejezchleb